storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/core/RocksDbInstanceFactory.java

```diff
@@ -112,6 +112,7 @@ private static DBOptions createDBOptions(final RocksDbConfiguration configuratio
         .setBytesPerSync(1048576L)
         .setWalBytesPerSync(1048576L)
         .setMaxBackgroundFlushes(2)
+        .setDbWriteBufferSize(configuration.getCacheCapacity())
```

Comment:
I did some playing just to see what happens if we make it higher. I think around 128MB has a good mix of the disk being used, might be a better starting point than 8MB...

give `.setDbWriteBufferSize(134217738L)` a go...

Reply:
@rolfyone thanks for this investigation! Did it 256M by default